### ISSUE TITLE
Custom loss function in pyspark

### ISSUE THREAD

**Issue by adamomainz** [catboost/catboost#1582]
Problem: No known/documented way of using custom loss functions in pyspark. Is there a way to go about this?
catboost version: 0.24.4
Operating System: linux (pyspark 2.24.0 python 3.6)

**Comment by kizill** [comment]
Hello! We are planning to refactor our custom loss function mechanics in #398

**Comment by adamomainz** [comment]
Thank you for the update! Would you be able to address https://github.com/catboost/catboost/issues/1579 and/or https://github.com/catboost/catboost/issues/1580 as part of https://github.com/catboost/catboost/issues/398 ?

**Comment by kizill** [comment]
Yes, it's a great idea, we'll try to find a contributor for those tasks.
